src/ralph/reports/forms.py

```diff
@@ -0,0 +1,15 @@
+from collections import Counter
+
+from django import forms
+from django.contrib.admin.options import BaseInlineFormSet
+from django.utils.translation import ugettext_lazy as _
+
+
+class ReportTemplateFormset(BaseInlineFormSet):
+    def clean(self):
+        defaults = [
```

Comment:
now it works. ``self.cleaned_data`` return list of ``clenead_data`` of each form in formset